### PR TITLE
Add linter to travis script

### PR DESCRIPTION
**Pull request by kalebdavis**

### CODE REVIEW

File: server/tests/routes/ticket.test.js [hunk 1]
```diff
@@ -136,8 +136,8 @@ describe('## Ticket APIs', () => {
   describe('# GET /api/v1/ticket/all?isOpen=...&groupName=...&to=...&from=...', () => {
     beforeEach((done) => {
       const promiseChain = [];
-      for (var i = 0; i < 3; i++) {
```
Comment (jarrydlee) [inline server/tests/routes/ticket.test.js]: Do you think we should just remove the no ++ rule?

File: server/tests/routes/user.test.js [hunk 2]
```diff
@@ -223,12 +223,12 @@ describe('## User APIs', () => {
     it('should update a device', (done) => {
       request(app)
         .put(`${userUrl}/${user.id}/device/${device.id}`)
-        .send(updateDetails)
+        .send(newDevice)
```
Comment (jarrydlee) [inline server/tests/routes/user.test.js]: Why `newDevice`? We are updating an old one.

File: server/tests/services/auth.test.js [hunk 3]
```diff
@@ -50,7 +50,7 @@ describe('## Auth Service', () => {
     });
 
     it('should not create a user that already exists', (done) => {
-      const user = {
+      user = {
```
Comment (jarrydlee) [inline server/tests/services/auth.test.js]: Shouldn't this have const?

Comment (kalebdavis) [inline server/tests/services/auth.test.js]: It's defined in an earlier scope

File: server/tests/routes/ticket.test.js (hunk 1 above)
Comment (kalebdavis) [inline server/tests/routes/ticket.test.js]: I'm ambivalent really

**Pull request merged**